src/index.d.ts

```diff
@@ -1,8 +1,9 @@
 import { ReactElement, Component } from 'react';
-import { ImageSourcePropType, NativeScrollEvent, NativeSyntheticEvent, TextStyle, ViewStyle, ImageResizeMode } from 'react-native';
+import { ImageSourcePropType,HeaderTypeProp, ScrollView, NativeScrollEvent, NativeSyntheticEvent, TextStyle, ViewStyle, ImageResizeMode } from 'react-native';
```

Comment:
pls add space after comma `ImageSourcePropType,HeaderTypeProp` to `ImageSourcePropType, HeaderTypeProp`

Reply:
done